underscore.js

```diff
@@ -1279,9 +1288,7 @@
 
   // Is a given value an array?
   // Delegates to ECMA5's native Array.isArray
-  _.isArray = nativeIsArray || function(obj) {
-    return toString.call(obj) === '[object Array]';
-  };
+  _.isArray = isArray;
```

Comment:
Unfortunately this time it _is_ getting called right away. The outer `_.property` function (which uses `isArray`) gets called when we create `getLength`.